lib/index.js

```diff
@@ -3,23 +3,39 @@
  */
 'use strict';
 
-const {documentLoader} = require('bedrock-jsonld-document-loader');
-const jsigs = require('jsonld-signatures');
```

Comment:
can `jsonld-signatures` be removed as a dependency in this module now?

Reply:
Yes, it's been removed now. Previously it was being used to get the Ed25519Signature2018 suite and now we are using Ed25519Signature2020 suite which we're getting from its own module.

Reply:
I just need to update the changelog entry.